### DISCUSSION TITLE
Adding kart info

### DISCUSSION THREAD

**Discussion by davidhcefx** [davidhcefx/Tracks-in-KartRider-Rush-Story-Mode#3] [Ideas]
I think it might also be beneficial to include kart information to our list. For example, the kart of **30-1** is *White Knight*.

<img src="https://krrplus.web.app/assets/Karts/r_0129_icon.png" alt="White Knight" width=130px>

To achieve this, we first need to know

1. which kart each chapter has, and then 
2. get a [list of kart names](https://krrplus.web.app/karts)

Maybe I'll try it someday when I have time lol (or does someone willing to help?)

**Comment by davidhcefx** [comment]
Update: We have already added some (68e57de), while there are more to be added.

**Reply by FauxBlue** [reply]
Worth noting that the linked kart list includes the literal translations of the kart names in brackets (for some karts).
e.g. Beetle Dasher (Black Beetle). The localised English name is "Beetle Dasher", whereas the Chinese name translates to "Black Beetle".
